collision_planner:
  ros__parameters:
    sim: False